sites/org/pages/docs/[...slug].mjs

```diff
@@ -219,9 +38,7 @@ export async function getStaticProps({ locale, params }) {
  * To learn more, see: https://nextjs.org/docs/basic-features/data-fetching
  */
 export async function getStaticPaths() {
-  const somePaths = mdxPaths
-    .filter((path) => path.split('/').length < 5)
-    .filter((path) => path !== 'docs')
+  const somePaths = mdxPaths.filter((path) => path.split('/').length < 5 && path !== 'docs')
```

Comment:
I feel all docs should be pre-built.

Reply:
They are. I just combined your two existing filter calls into one filter call